javascripts/tagmanager.js

```diff
@@ -425,18 +427,18 @@
 
                     var start, end;
                     try {
-                        start = new Date(startDateTime);
+                        start = new Date(startDateTime + (startDateTime && startDateTime.split(' ').length === 2 ? ' UTC' : ''));
```

Comment:
It took me a while to understand this code and what it does. Maybe we can put this into a function and give a naming so it'll become more clear? Then we could also reuse it in the endDate below.

Part of it why it was hard to understand was because it wasn't clear what `startDateTime` looks like. After looking at tests I figured it must be `2022/01/01 06:00:00` meaning if no timezone is added it will assume the value is in UTC.

Reply:
@tsteur changed to function, I was wondering should we also update the description for start and end date ?
![Screenshot from 2022-08-17 12-09-09](https://user-images.githubusercontent.com/14902277/185051718-880fd739-a2a0-4733-a523-5cb15432bb56.png)

Change `If set, this tag will be fired only after this start date. The specified date will be assumed to be in your local timezone.
Your current local time is Wed Aug 17 2022 12:10:31 GMT+0530 (India Standard Time).`

to 
`If set, this tag will be fired only after this start date. The specified date will be converted to UTC datetime for comparison.
Your current local time is Wed Aug 17 2022 12:10:31 GMT+0530 (India Standard Time).`

Reply:
> The specified date will be converted to UTC datetime for comparison.

I wouldn't add it as it might confuse people. They might not know what UTC is and they may be confused why we mention it since they wouldn't need to know that. It's maybe a technical implementation detail?

Reply:
Nice on adding the function 👍  Now we could actually even add a simple test for it? That would also make it easier for other people to understand what the function does when they come across it. A simple basic test or two could do

Reply:
@tsteur added, please check